docs/src/pages/[platform]/components/phonenumberfield/examples/PhoneNumberFieldThemeExample.tsx

```diff
@@ -12,7 +12,7 @@ const theme: Theme = {
   },
 };
 
-export const ThemingExample = () => (
```

Comment:
We need to update the example once tokens are added. The example will mislead users because this will override all field controls' styling.